phoenix-core-client/src/main/java/org/apache/phoenix/query/ConnectionQueryServicesImpl.java

```diff
@@ -774,6 +822,20 @@ public void close() throws SQLException {
         }
     }
 
+    // Based on org.apache.hadoop.hbase.client.ConnectionImplementation
+    private void shutdownThreadPool(ThreadPoolExecutor pool) {
+        if (pool != null && !pool.isShutdown()) {
+            pool.shutdown();
+            try {
+                if (!pool.awaitTermination(10, TimeUnit.SECONDS)) {
```

Comment:
You want leave the wait time to be 60 sec? (to match the HTable pool shutdown logic)